File: MachineLearning/Udemy/Preprocessing/librerias.R
#Plantilla para el pre procesado de datos
#importar el dataset
dataset = read.csv('Data.csv')



#dividir los datos en conjunto de entrenamiento y conjunot de test
#install.packages("caTools")
set.seed(123)
split = sample.split(dataset$Purchased, SplitRatio = 0.8)
train_set = subset(dataset, split==TRUE)
test_set = subset(dataset, split==FALSE)
#escalado de valores

#train_set[,2:3] = scale(train_set[,2:3])
#test_set[,2:3] = scale(test_set[,2:3])

File: MachineLearning/Udemy/Preprocessing/librerias.py
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from sklearn.impute import SimpleImputer  # calculo de los datos
from sklearn import preprocessing
#importar el dataset

dataset = pd.read_csv('Data.csv')
x = dataset.iloc[:,0:3].values  #localizar elementos por posicion
#x = dataset.iloc[:,:-1].values
y = dataset.iloc[:,3].values


#DIVIDIR el dataset en conjunto de entrenamiento y conjunto de testing
from sklearn.model_selection import train_test_split
x_train , x_test , y_train, y_test =  train_test_split(x,y,test_size=0.2,random_state=0)

# escalado de variables
"""sc_x= preprocessing.StandardScaler()
x_train = sc_x.fit_transform(x_train)
x_test = sc_x.transform(x_test)"""